Data_Analysis_Markdown.Rmd

```diff
@@ -0,0 +1,503 @@
+frame[[name]]$variable.order <- factor(frame[[name]]$variable, levels = c("Female", 
+                                                                          "Age", 
+                                                                          "Urban", 
+                                                                          "Parental Occupation", 
+                                                                          "Skin Color"))
+  
+country.fig[[name]] <- ggplot(frame[[name]], aes(y = coefficient, x = variable.order)) + geom_point() +
+  geom_pointrange(aes(ymin = ci.lower,  ymax = ci.upper)) +
+  geom_hline(yintercept = 0, color = "dark green", size = 1) +
+  ylim(-.5, .5) +
+  coord_flip() +
+  ylab("") + xlab("") +
+  theme_classic() +
+  theme(plot.title = element_text(lineheight = .6, face = "bold"),
+        axis.text = element_text(colour = "black", face = "bold"), 
+        axis.title = element_text(face = "bold"))
+
+}
+
+country.fig.note.brazil <- arrangeGrob(country.fig[["Brazil"]], 
+                         sub = textGrob("Brazil",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.mexico <- arrangeGrob(country.fig[["Mexico"]], 
+                         sub = textGrob("Mexico",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.guatemala <- arrangeGrob(country.fig[["Guatemala"]], 
+                         sub = textGrob("Guatemala",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.colombia <- arrangeGrob(country.fig[["Colombia"]], 
+                         sub = textGrob("Colombia",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.ecuador <- arrangeGrob(country.fig[["Ecuador"]], 
+                         sub = textGrob("Ecuador",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.bolivia <- arrangeGrob(country.fig[["Bolivia"]], 
+                         sub = textGrob("Bolivia",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.peru <- arrangeGrob(country.fig[["Peru"]], 
+                         sub = textGrob("Peru",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.dominican_republic <- arrangeGrob(country.fig[["Dominican Republic"]], 
```

Comment:
Was thinking here too that a function might be helpful to avoid repetition. (Something I need to do for parts of my code as well!)